rundeckapp/grails-app/services/rundeck/services/logging/ThresholdLogWriter.groovy

```diff
@@ -20,33 +20,55 @@ import com.dtolabs.rundeck.core.logging.FilterStreamingLogWriter
 import com.dtolabs.rundeck.core.logging.LogEvent
 import com.dtolabs.rundeck.core.logging.LogUtil
 import com.dtolabs.rundeck.core.logging.StreamingLogWriter
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import rundeck.Execution
 
 import java.util.concurrent.atomic.AtomicBoolean
 
 /**
  * Used to log that a threshold was reached, and optionally truncate further output
  */
 class ThresholdLogWriter extends FilterStreamingLogWriter {
+    static final Logger LOG = LoggerFactory.getLogger(ThresholdLogWriter.class)
+
+    static final String MSG_WARN_SIZE = "The log file size achieved more than {} bytes - current size: {} bytes"
+
     LoggingThreshold threshold
     final boolean truncate
+    Execution execution
```

Comment:
it seems the only use for this `execution` is retrieving the job name.  It would be better to pass in a String to include in the warning message at this location https://github.com/rundeck/rundeck/pull/7199/files#diff-784bda3682bff4a5c6ff2ed5b7db3bd2676701746ed28e6a3fbfd82244a7b185R184

Reply:
Done